pkgs/development/tools/misc/terraform-plugin-docs/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "terraform-plugin-docs";
+  version = "0.16.0";
+
+  src = fetchFromGitHub {
+    owner = "hashicorp";
+    repo = "terraform-plugin-docs";
+    rev = "v${version}";
+    sha256 = "1nwvi3nb4hxpa8pz0zm41yp7skvfafrl313n6zplr0lcs7my5xp6";
+  };
+
+  subPackages = [ "cmd/tfplugindocs" ];
+  vendorSha256 = "AjW6BokLVDkIWXToJ7wNq/g19xKTAfpQ/gVlKCV5qw0=";
+
```

Comment:
What do you think about passing the version number in the ldlfags:

```nix
  ldflags = [ "-s" "-w" "-X main.version=${version}" "-X main.commit=${src.rev}" ];
```

so that tfplugindocs returns the correct version number:

```console
$ tfplugindocs --version
tfplugindocs Version 0.16.0 from commit v0.16.0
```

instead of:

```console
$ tfplugindocs --version
tfplugindocs Version dev
````

?